test/framework/kogitoserviceutils.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/meta"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"github.com/kiegroup/kogito-cloud-operator/test/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// WaitForService waits that the service has a certain number of replicas
+func WaitForService(namespace string, serviceName string, replicas int, timeoutInMin int) error {
+	return WaitForOnOpenshift(namespace, serviceName+" running", timeoutInMin,
+		func() (bool, error) {
+			deployment, err := GetDeployment(namespace, serviceName)
+			if err != nil {
+				return false, err
+			}
+			if deployment == nil {
+				return false, nil
+			}
+			return deployment.Status.Replicas == int32(replicas) && deployment.Status.AvailableReplicas == int32(replicas), nil
+		})
+}
+
+// CustomResourceBuilder helps to configure a CR resource used by the InstallService method.
+type CustomResourceBuilder struct {
```

Comment:
Can't we reuse v1alpha1.KogitoServcie interface ?
https://github.com/kiegroup/kogito-cloud-operator/blob/4d9ec51b70ea1abdfb7ffc65c9be124e706fb34c/pkg/apis/app/v1alpha1/kogitoservices_types.go#L25

Reply:
Done. Thanks for the tip!